com.github.muriloventuroso.pdftricks.json

```diff
@@ -75,9 +81,9 @@
                 "/bin/unix-lpr.sh"
             ],
             "sources": [{
-                    "type": "archive",
-                    "url": "https://github.com/ArtifexSoftware/ghostpdl-downloads/releases/download/gs9540/ghostscript-9.54.0.tar.xz",
-                    "sha256": "c2b7b43cde600f4e70efb2cd95865f6d884a67315c3de235914697d8ccde6e3b"
+                    "type": "git",
+                    "url": "https://github.com/ArtifexSoftware/ghostpdl",
+                    "tag": "ghostpdl-10.05.0"
```

Comment:
why switching to git? Please still use tarballs.

Reply:
was easier to find the new tag there, but ok.